docs/data/data-grid/export/export.md

```diff
@@ -16,14 +16,14 @@ See the [Export component](/x/react-data-grid/components/export/) for examples o
 
 ## Export options
 
-By default, the toolbar displays all the available export formats, according to your license, which are
+Following are the available export options:
 
 - [Print](#print-export)
 - [CSV](#csv-export)
+- [Clipboard](#clipboard)
```

Comment:
Clipboard [export](https://mui.com/x/react-data-grid/clipboard/#clipboard-copy) is MIT, it's clipboard [import](https://mui.com/x/react-data-grid/clipboard/#clipboard-paste) that is Premium.